### PR TITLE
[13.0][MIG] sale_order_import

### PR DESCRIPTION
**Pull request by sebalix**
Based on 12.0 migration PR: https://github.com/OCA/edi/pull/101

Depends on:
- [x] https://github.com/OCA/edi/pull/200
- [x] https://github.com/OCA/sale-workflow/pull/1023

### CODE REVIEW

File: sale_order_import/models/sale.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+from odoo import _, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def name_get(self):
+        """Add amount_untaxed in name_get of sale orders"""
+        res = super(SaleOrder, self).name_get()
```
Comment (i-vyshnevska) [inline sale_order_import/models/sale.py]:
```suggestion
        res = super().name_get()
```

File: sale_order_import/models/sale.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        if self._context.get("sale_order_show_amount"):
+            new_res = []
+            for (sale_id, name) in res:
+                sale = self.browse(sale_id)
+                # I didn't find a python method to easily display
+                # a float + currency symbol (before or after)
+                # depending on lang of context and currency
```
Comment (i-vyshnevska) [inline sale_order_import/models/sale.py:19]: this looks wierd, symboll is extracted in the view dynamically

File: sale_order_import/wizard/sale_order_import.py [hunk 3]
```diff
@@ -0,0 +1,587 @@
...
+    # for state = update
+    commercial_partner_id = fields.Many2one(
+        "res.partner", string="Commercial Entity", readonly=True
+    )
+    partner_shipping_id = fields.Many2one(
+        "res.partner", string="Shipping Address", readonly=True
+    )
+    # amount_untaxed = fields.Float(
+    #    string='Total Untaxed', digits=dp.get_precision('Account'),
+    #    readonly=True)
```
Comment (i-vyshnevska) [inline sale_order_import/wizard/sale_order_import.py]: ?

File: sale_order_import/models/sale.py (hunk 2 above)
Comment (sebalix) [reply sale_order_import/models/sale.py:19]: Yes, this comment is dated from 2018, but we could let it as a reminder as having a method doing that server-side could be nice (maybe it exists now?).

Comment (sebalix) [reply sale_order_import/models/sale.py:19]: I only found `amount_to_text`, but it is not doing what we want :)
```python
>>> env.ref("base.EUR").amount_to_text(100)                                                                                         
'One Hundred Euros'
```

**Comment by sebalix** [comment]
Rebased to get Travis green. Don't know why runbot is still red, it seems it is actually running.

**Review by jgrandguillaume — APPROVED** [review]
LGTM

**Pull request merged**